frame/layer.go

```diff
@@ -92,8 +92,12 @@ func (l *Layer) bgWork() {
 				if err != nil {
 					l.l.Error("send-nak-failed", zap.Error(err))
 				}
-			} else {
-				// @todo handle timeout(?)
+			} else if frameIn.status == ParseTimeout {
+				l.l.Warn("receive frame timeout")
+				err := l.sendNak()
```

Comment:
Still trying to figure out if the `nak` frame is necessary here or not. May leave it out unless we find a way to test if it’s necessary

Reply:
Is there a spec link you can provide for this as well? Curious if the spec says you MUST send a nak.

Reply:
@zacatac Updated to remove NAK. This assumes my reading of the manual is correct and matches what's being handled here (both questionable).

In any case, this just means we log something interesting if we encounter this case.